website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,496 @@
+---
+title: Customize dbt models database, schema, and alias
+id: customize-schema-alias
+description: "Learn how to properly adjust your generate_schema_name() and generate_alias_name() macros."
+displayText: Learn how to adjust your generate schema name and generate alias name.
+hoverSnippet: Learn how to adjust your generate schema name and generate alias name.
+# time_to_complete: '30 minutes' commenting out until we test
+icon: 'guides'
+hide_table_of_contents: true
+level: 'Advanced'
+recently_updated: true
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+This guide explains how to customize the [schema](/docs/build/custom-schemas), [database](/docs/build/custom-databases), and [alias](/docs/build/custom-aliases) naming conventions in dbt to fit your data warehouse governance and design needs.
+When we develop dbt models and perform a `dbt run` or `dbt build` command, objects (like tables and views) get created in the data warehouse based on these naming conventions.
+
+In the rest of the article, for consistency, when we refer to database, we'll be referring to a Google Cloud Platform  project or to a Databricks catalog.
+:::info A word on naming
+
+Different warehouses have different names for _logical databases_. The information in this document covers "databases" on Snowflake, Redshift, and Postgres; "projects" on BigQuery; and "catalogs" on Databricks Unity Catalog.
+
+:::
+The following is dbt's out-of-the-box default behavior:
+
+- The database where the object is created is defined by the database configured at the [environment level in dbt Cloud](/docs/dbt-cloud-environments) or in the [`profiles.yml` file](/docs/core/connect-data-platform/profiles.yml) in dbt Core.
+
+- The schema depends on whether you have defined a [custom schema](/docs/build/custom-schemas) for the model:
```

Comment:
Can you adjust the tabs and fix the ordering of this section? 

- The schema depends on whether you have defined a [custom schema](/docs/build/custom-schemas) for the model:
  - If you haven't defined a custom schema, dbt creates the object in the default schema. In dbt Cloud this is typically `dbt_username` for development and the default schema for deployment environments. In dbt Core, it uses the schema specified in the `profiles.yml` file.
  - If you define a custom schema, dbt concatenates the schema mentioned earlier with the custom one.
  - For example, if the configured schema is `dbt_myschema` and the custom one is `marketing`, the objects will be created under `dbt_myschema_marketing`.
   - Note that for automated CI jobs, the schema name derives from the job number and PR number: `dbt_cloud_pr_<job_id>_<pr_id>[_custom_schema_name]`
 
 ... the rest looks right!